### PR TITLE
Accounts: Add caveats about consumption

### PR DESCRIPTION
**Pull request by rhetoric101**

### CODE REVIEW

**Comment by paperclypse** [comment]
@rhetoric101 I missed this one. Do you still need a review/approval on it?

**Comment by rhetoric101** [comment]
@paperclypse I am still waiting on an SME for approval.

**Comment by paperclypse** [comment]
@rhetoric101 Let's close this one for now and re-open it when the SME gets back to you.

**Comment by rhetoric101** [comment]
@paperclypse I received approval from the SMEs. Could you review and approve it?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx [hunk 1]
```diff
@@ -16,11 +16,20 @@ redirects:
...
+* The results are approximate and don't correspond exactly to the values used to calculate your invoice amount. 
+* Use these tools to find and track areas of high usage.
+* Use queries to create alerts so you know if you're approaching a limit.
 
 ## Data ingest usage queries [#data-queries]
 
-The [usage UI](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing#usage-ui) displays your data ingest over time. If you need more detail than the UI provides, you can use NRQL queries like these: 
+The [usage UI](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing#usage-ui) displays your data ingest over time. If you need more detail than the UI provides, you can use a variety of NRQL queries. The examples below include two data types:
+
+* `NrConsumption`: This is useful for diving deep into the hourly details of usage. It's less helpful as a proxy for your bill.
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx:31]:
```suggestion
* `NrConsumption`: Use this to dive deep into the details of your usage hour-by-hour. It's less helpful as a proxy for your bill.
```

How about this?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx [hunk 2]
```diff
@@ -16,11 +16,20 @@ redirects:
...
+* `NrMTDConsumption`: This has a monthly aggregation of usage that gives you a good approximation of usage.
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx:32]:
```suggestion
* `NrMTDConsumption`: Use this to see a good approximation of your monthly aggregated usage.
```

How about this?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 3]
```diff
@@ -31,18 +31,17 @@ Some older New Relic customers are on our original pricing model. For more on th
...
 
-## The billing and usage UI [#usage-ui]
+## See your usage in the UI [#usage-ui]
 
-If you're a New Relic customer, the best way to understand your billing is to go to the [usage UI page](https://one.newrelic.com/admin-portal/consumption-usage/home). To find that, go to **[one.newrelic.com](https://one.newrelic.com)**, click the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings), then click **Administration**. You'll land on the **Plan & usage** UI page, which shows an overview of your usage.
+If you're a New Relic customer, you can see your usage on our **Plan & usage** page. While the numbers are an approximation, you can use those views to get a general idea about how your consumption might affect your bill. 
+
+To find the usage UI, go to **[one.newrelic.com](https://one.newrelic.com)**, click the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings), and then click **Administration**. You'll land on the **Plan & usage** UI page, which shows an overview of your usage like this:
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:38]:
```suggestion
To find the usage UI, go to **[one.newrelic.com](https://one.newrelic.com)**, click the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings), and then click **Administration**. The **Plan & usage** UI page shows an overview of your usage like this:
```

How about this?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx [hunk 4]
```diff
@@ -16,11 +16,20 @@ redirects:
...
+To help you understand your [New Relic usage and billing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), we provide a [usage UI](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing#usage-ui) for understanding your billing-related usage. Also, you can run custom NRQL queries to get additional detail, and you can create alert notifications for when your usage has reached certain levels. 
+
+Keep the following in mind when using the usage UI or making consumption queries:
+
+* The results are approximate and don't correspond exactly to the values used to calculate your invoice amount. 
+* Use these tools to find and track areas of high usage.
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx:24]:
```suggestion
* Use these tools to find and track your areas of high usage.
```

**Review by paperclypse — APPROVED** [review]
I left some comments for you to consider.

**Review by paperclypse — APPROVED** [review]
Looks good!

**Pull request merged**